src/main/java/com/example/survey/models/TextQuestion.java

```diff
@@ -0,0 +1,35 @@
+package com.example.survey.models;
+
+import javax.persistence.*;
+
+@Entity
+public class TextQuestion {
+    private String question;
```

Comment:
Can we use inheritance for these please?

Reply:
I wanted to make a abstract `Question` class but couldnt figure out how to make it work with jpa

Reply:
Good enough for me